docusaurus/docs/learn/quickstarts/network/miniziti.bash

```diff
@@ -136,6 +182,14 @@ logInfo() {
     echo "INFO: $*"
 }
 
+logWarn() {
+    (( $# )) || {
+        logError "logWarn() takes 1 or more args"
+        return 1
+    }
+    echo "WARN: $*"
```

Comment:
```suggestion
    echo "WARN: $*" >&2
```

Reply:
Fixed. This function was missing but was being called during delete, so it wasn't an attempt at an original implementation, but a copy-paste of the info variant.

Reply:
What is a "kill switch var"?

Reply:
I meant the global vars in the functions that are tested when the function is run. 

If they evaluate true then the function runs, else the function returns an error. 

Normally, I was surprised to find a "kill switch" like that inside the function. I expected it to check whether the function should run before calling it.

Reply:
Updated to make the check in the caller. FWIW, calling something from a `main()` function and having it check the global there is not really any different. I used `if` instead of `&&` here due to issues with syntax highlighters in nested statements with the following shape:

```bash
(( test1 )) && {
  (( test2 )) && statement
}
```
Originally, this was the movitivation for moving them into the function in the first place.